config/analytics.yml

```diff
@@ -0,0 +1,303 @@
+    - ukprn
+    - last_changed_date
+    - address_1
+    - address_2
+    - address_3
+    - town
+    - county
+    - postcode
+    - easting
+    - northing
+    - region
+    - country
+    - created_at
+    - updated_at
+    - establishment_type_code
+    - establishment_type_name
+    - high_pupil_premium
+    - postcode_without_spaces
+    - number_of_pupils
+    - eyl_funding_eligible
+    - phase_type
+    - phase_name
+  :schedules:
+    - id
+    - course_group_id
+    - cohort_id
+    - name
+    - identifier
+    - applies_from
+    - applies_to
+    - allowed_declaration_types
+    - created_at
+    - updated_at
+    - ecf_id
+  :registration_interests:
+    - id
+    - email
+    - notified
+    - created_at
+    - updated_at
+  :private_childcare_providers:
+    - id
+    - provider_urn
+    - provider_name
+    - registered_person_urn
+    - registered_person_name
+    - registration_date
+    - provider_status
+    - address_1
+    - address_2
+    - address_3
+    - town
+    - postcode
+    - postcode_without_spaces
+    - region
+    - local_authority
+    - ofsted_region
+    - early_years_individual_registers
+    - provider_early_years_register_flag
+    - provider_compulsory_childcare_register_flag
+    - places
+    - created_at
+    - updated_at
+    - disabled_at
+  :participant_outcome_api_requests:
+    - id
+    - ecf_id
+    - participant_outcome_id
+    - request_path
+    - status_code
+    - request_headers
+    - request_body
+    - response_body
+    - response_headers
+    - created_at
+    - updated_at
+  :participant_outcomes:
+    - id
+    - state
+    - completion_date
+    - declaration_id
+    - qualified_teachers_api_request_successful
+    - sent_to_qualified_teachers_api_at
+    - created_at
+    - updated_at
+    - ecf_id
+  :participant_id_changes:
+    - id
+    - user_id
+    - created_at
+    - updated_at
+    - ecf_id
+    - from_participant_id
+    - to_participant_id
+  :local_authorities:
+    - id
+    - ukprn
+    - name
+    - address_1
+    - address_2
+    - address_3
+    - town
+    - county
+    - postcode
+    - postcode_without_spaces
+    - high_pupil_premium
+    - created_at
+    - updated_at
+  :lead_providers:
+    - id
+    - name
+    - created_at
+    - updated_at
+    - ecf_id
+    - hint
+  :itt_providers:
+    - id
+    - legal_name
+    - operating_name
+    - removed_at
+    - approved
+    - created_at
+    - updated_at
+    - disabled_at
+  :declarations:
+    - id
+    - ecf_id
+    - application_id
+    - superseded_by_id
+    - lead_provider_id
+    - cohort_id
+    - declaration_type
+    - declaration_date
+    - state
+    - state_reason
+    - created_at
+    - updated_at
+  :course_groups:
+    - id
+    - name
+    - created_at
+    - updated_at
+  :courses:
+    - id
+    - name
+    - created_at
+    - updated_at
+    - ecf_id
+    - description
+    - position
+    - display
+    - identifier
+    - course_group_id
+  :contract_templates:
```

Comment:
are we sure we want to send this across, in ECF we avoided sending payment data?